RichEditorView/Assets/editor/rich_editor.js

```diff
@@ -48,19 +49,19 @@ RE.rangeOrCaretSelectionExists = function() {
 };
 
 RE.editor.addEventListener("input", function() {
-    RE.updatePlaceholder();
-    RE.backuprange();
-    RE.callback("input");
-});
+                           RE.updatePlaceholder();
+                           RE.backuprange();
+                           RE.callback("input");
+                           });
```

Comment:
Same for all these

Reply:
got those fixed:
![Screen Shot 2019-10-03 at 6 25 28 PM](https://user-images.githubusercontent.com/50113568/66170812-5ebd3e00-e60b-11e9-965f-486be9751c3f.png)